### ISSUE TITLE
SuperScript should work on Windows.

### ISSUE THREAD

**Issue by silentrob** [superscriptjs/superscript#274]
^ Lets test superscript for windows compatibility.

**Comment by OmarShehata** [comment]
If it helps, I can tell you that right off the bat I get “Failed to locate: CL.exe" as described [here](http://stackoverflow.com/questions/33239445/could-not-install-prerender-using-npm-failed-to-locate-cl-exe).

I don't have a lot of experience with Visual Studio so I didn't try too hard to get it to work, so I stopped there and switched to trying to get it to work with Vagrant.

**Comment by nsaraiya** [comment]
Hi @silentrob - can you update node-normalizer to use google/re2 directly instead of uhop/node-re2? node-re2 causes npm install superscript to fail on Windows. I also checked - node-re2 uses a google/re2 version published 2 years ago whereas the google/re2 code base is very actively managed and gets updated every few days.

**Comment by fmtoffolo** [comment]
@nsaraiya where you able to install it? I'm having the same problem with the re2 build.

**Comment by nsaraiya** [comment]
No I had to pause at this step. This cannot be fixed until node-normalizer changes to use the correct re2 version.

**Comment by fmtoffolo** [comment]
So it is basically uninstalable on windows until a third party dependency gets fixed. I should install Linux...

**Comment by slidenerd** [comment]
29 days ago I see that you have commited something about RE2, does that mean this works on Windows now?

**Comment by ianp** [comment]
I'm working on getting the third party dev (`node-re2`) updated now, hopefully should be done in the next few days.

**Comment by joebain** [comment]
I've just been trying out superscript, and I'm using the new [bash on windows](https://msdn.microsoft.com/en-us/commandline/wsl/about) tools. I installed superscript successfully and ran the "hello-superscript" test bot. I had to make a small adjustment to the log file writing code in user.js ([pull request](https://github.com/superscriptjs/superscript/pull/296)) to get it working. I also found the windows telnet client doesn't work, it seems to send data on every keypress, rather than at a newline as the client expects. This may be a discrepancy in node though, I'm not sure. I was able to use the bash on windows telnet client successfully.

Hope this helps someone out there, I'm looking forward to trying out more of superscript!
